internal/bgp/frrk8s/frrk8s.go

```diff
@@ -48,13 +47,23 @@ type session struct {
 	logger         log.Logger
 }
 
-// sessionName() defines the format of the key of the 'sessions' map in
-// the 'frrState' struct.
+// sessionName() defines the format of a map key.
 func sessionName(s session) string {
-	baseName := fmt.Sprintf("%d@%s-%d@%s", s.PeerASN, s.PeerAddress, s.MyASN, s.SourceAddress)
+	asn := strconv.FormatUint(uint64(s.PeerASN), 10)
+	if v := s.DynamicASN; v != "" {
+		asn = v
+	}
```

Comment:
nit: `if s.DynamicASN != ""` and `asn = s.DynamicASN` looks cleaner

Reply:
(not sure) but done